scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc

```diff
@@ -13,9 +13,7 @@ include::modules/cnf-about_hyperthreading_for_low_latency_and_real_time_applicat
 [role="_additional-resources"]
 .Additional resources
 
-* xref:../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#configuring_hyperthreading_for_a_cluster_{context}[Configuring hyperthreading for a cluster]
-
-include::modules/cnf-installing-the-performance-addon-operator.adoc[leveloffset=+1]
+* xref:../scalability_and_performance/cnf-low-latency-tuning.adoc#configuring_hyperthreading_for_a_cluster_{context}[Configuring hyperthreading for a cluster]
 
 include::modules/cnf-upgrading-performance-addon-operator.adoc[leveloffset=+1]
```

Comment:
wasn't this section removed ?

Reply:
That entire assembly was removed/renamed.